### PR TITLE
Feat/register form

### PR DESCRIPTION
**Pull request by kurilyov**

### CODE REVIEW

**Review by mayst — CHANGES_REQUESTED** [review]
make validation for reg form. move it to page folder. make general components for textfield and btn

File: src/components/FormInput.tsx
Comment (mayst) [inline src/components/FormInput.tsx:1]: make a folder 'inputs' put this component there. rename it to 'TextFieldBx'

Comment (kurilyov) [reply src/components/FormInput.tsx:1]: Done

**Comment by kurilyov** [comment]
Done

**Review by mayst — APPROVED** [review]

**Pull request merged**